src/sections/header.html

```diff
@@ -1,4 +1,42 @@
-<div>
-    <h1 class="text-blue-800">Header</h1>
-    <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Quisquam, quibusdam.</p>
-</div>
\ No newline at end of file
+<header class="md:flex justify-between items-center md:p-6">
+        <div class="hidden md:block">
+            <a href="#" class="block font-semibold">DV</a>
+            <a href="#" class="block font-semibold">EN</a>
```

Comment:
I don't know why we all didn't catch this one, but it's supposed to be:
```suggestion
            <a href="#" class="block font-semibold">DE</a>
            <a href="#" class="block font-semibold">EN</a>
```